tutorials/holohub_operators_external_applications/holohubCLI_wrapper

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+# SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+SCRIPT_PATH="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+# Pulls the holohub CLI from the holohub repo
+if [ ! -d "${SCRIPT_PATH}/utilities/cli" ]; then
+    git clone --depth 1 --no-checkout https://github.com/nvidia-holoscan/holohub.git ${SCRIPT_PATH}/.tmp-holohub/holohub \
+        && cd ${SCRIPT_PATH}/.tmp-holohub/holohub \
+        && git sparse-checkout init --cone \
+        && git sparse-checkout set utilities/cli utilities/metadata utilities/testing cmake\
+        && git checkout main \
+        && mv ${SCRIPT_PATH}/.tmp-holohub/holohub/utilities ${SCRIPT_PATH} \
+        && mv ${SCRIPT_PATH}/.tmp-holohub/holohub/cmake ${SCRIPT_PATH} \
+        && rm -rf ${SCRIPT_PATH}/.tmp-holohub \
+        && cd ${SCRIPT_PATH}
+fi
+
+# Check if RTI_LICENSE_FILE is set
+if [ -n "$RTI_LICENSE_FILE" ]; then
+    if [ "$1" == "run" ] && [[ ! " $@ " =~ " --local " ]]; then
+      DOCKER_OPTS="--docker-opts=-v ${RTI_LICENSE_FILE}:/workspace/rti_license.dat"
+    fi
+fi
+
+# TODO: Set the command name to the script name
+export HOLOHUB_CMD_NAME='./holohubCLI_wrapper'
+
+# Only pass DOCKER_OPTS if it's set and not empty
+if [ -n "$DOCKER_OPTS" ]; then
+    PYTHONPATH=${PYTHONPATH}:${SCRIPT_PATH} python3 ${SCRIPT_PATH}/utilities/cli/holohub.py "$@" "$DOCKER_OPTS"
+else
+    PYTHONPATH=${PYTHONPATH}:${SCRIPT_PATH} python3 ${SCRIPT_PATH}/utilities/cli/holohub.py "$@"
+fi
```

Comment:
Suggest doing away with DOCKER_OPTS for DDS as an I4H fork implementation detail, can simply forward all args with "$@"